setup.py

```diff
@@ -45,10 +44,12 @@ def prerelease_local_scheme(version):
 sources = {
     'bioformats': [f'large-image-source-bioformats{limit_version}'],
     'deepzoom': [f'large-image-source-deepzoom{limit_version}'],
+    'dicom': [f'large-image-source-dicom{limit_version} ; python_version < "3.8"'],
```

Comment:
Shouldn't this be `python_version >= "3.8"`? That's what it is with what was removed.

As is, this means anyone above Python 3.7 cannot install this source

Reply:
Yep.  Fixed and testing.